src/components/ChooseCheckGroup/CheckGroupCard.tsx

```diff
@@ -99,10 +102,57 @@ function getTooltip(
 }
 
 const getStyles = (theme: GrafanaTheme2) => ({
+  newBadge: css({
+    position: 'absolute',
+    right: 0,
+    marginRight: theme.spacing(3),
+    marginTop: theme.spacing(2),
+    height: '26px',
+  }),
+
+  cardHeader: css({
+    maxHeight: '20px',
+  }),
```

Comment:
This isn't actually being used because of a bug in <Card.Heading> where the class isn't being passed (but probably a good indicator we can get rid of it).